tests/python/pants_test/goal/test_run_tracker.py

```diff
@@ -14,7 +14,7 @@
 
 
 class RunTrackerTest(TestBase):
-  def test_upload_stats(self):
+  def test_upload_stats(self, response_code=200):
```

Comment:
Couple nits:

1) Would be great to make this kwarg only, so that you have to specify `response_code` and can't use position args when calling:

```python
def test_upload_stats(self, *, response_code=200):
```

2) Type hints always appreciated. Even though these are intuitive enough, higher coverage makes MyPy work better and encourages others to write them.
```python
def test_upload_stats(self, *, response_code: int = 200) -> None:
```

3) This name doesn't work great with Pytest test discovery. Because it starts with `test_`, I believe Pytest will try to run it and will do so with the default arg. Instead, I recommend renaming this to something like `assert_upload_stats`.